.github/workflows/e2e_android.yml

```diff
@@ -99,8 +98,8 @@ jobs:
       - name: 'Get Matrix Value'
         id: matrix_value
         run: |
-          if [[ "${{ inputs.run_full_test }}" == "true" ]] || [[ ${{ github.event_name }} == 'schedule' ]] || [[ "${{ inputs.tests_to_run}}" != "" ]]; then
-            if [[ "${{ inputs.run_full_test }}" == "true" ]] || [[ ${{ github.event_name }} == 'schedule' ]]; then
+          if [[ "${{ inputs.run_full_test }}" == "true" ]] || [[ ${{ github.event_name }} == 'schedule' ]] || [[ "${{ inputs.tests_to_run}}" != "" ]] || [[ ${{ github.event_name }} == 'push' ]]; then
```

Comment:
Wouldn't this run all tests on branches anytime a push happens? If we only want to change the logic for just Dependabot PRs we could do something like

```yml
[[ "${{ github.event.pull_request.user.login }}" == "dependabot[bot]" ]]
```